pipeline.py

```diff
@@ -72,31 +89,115 @@ class TextProcessingPipeline:
     """
 
     def __init__(self, corpus_manager: CorpusManager):
-        pass
+        self.corpus_manager = corpus_manager
 
     def run(self):
         """
         Runs pipeline process scenario
         """
-        pass
+        for article in self.corpus_manager.get_articles().values():
+            tokenized_text = self._process(article.get_raw_text())
+
+            cleaned = []
+            single_tagged = []
+            multiple_tagged = []
+
+            for token in tokenized_text:
+                cleaned.append(token.get_cleaned())
+                single_tagged.append(token.get_single_tagged())
+                multiple_tagged.append(token.get_multiple_tagged())
+
+            article.save_as(' '.join(cleaned), ArtifactType.cleaned)
+            article.save_as(' '.join(single_tagged), ArtifactType.single_tagged)
+            article.save_as(' '.join(multiple_tagged), ArtifactType.multiple_tagged)
 
     def _process(self, raw_text: str):
         """
         Processes each token and creates MorphToken class instance
         """
-        pass
+        cleaned_text = raw_text.replace('-\n', '')
```

Comment:
why?

Reply:
I see. If we do not do it, we have gap between parts of one word, so Mystem/pymorphy processes not relevant for us tokens (parts of words) and gives us wrong information.